lib/pages/chat_details/chat_details_edit.dart

```diff
@@ -27,7 +27,7 @@ import 'package:linagora_design_flutter/images_picker/asset_counter.dart';
 import 'package:linagora_design_flutter/images_picker/images_picker_grid.dart';
 import 'package:matrix/matrix.dart';
 import 'package:flutter_gen/gen_l10n/l10n.dart';
-import 'package:photo_manager/photo_manager.dart';
+import 'package:photo_manager/photo_manager.dart' as photo_manager;
```

Comment:
why don't you expose the `AssetEntity` inside `linagora_design_flutter` package, so you don't need to have this import ?. I know it is the same, but the code is more readable. Thanks